gr-utils/python/modtool/modtool_add.py

```diff
@@ -26,17 +26,20 @@
 
 import os
 import re
+import click
+import sys
```

Comment:
PyLint will most probably complain about this import order. Which means that you could still probably lint all the files you modified.

Reply:
@primercuervo: Yes, this is incorrect. The standard imports should be placed first. Corrected.